src/Clock.tsx

```diff
@@ -0,0 +1,60 @@
+import { Component } from 'react';
+
+type Props = {
+  name: string,
+};
+
+type State = {
+  today: Date,
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      const today = new Date();
+
+      this.setState({ today });
+
+      // eslint-disable-next-line no-console
+      console.info(today.toUTCString().slice(-12, -4));
```

Comment:
It'll be cool to make function that makes string from Date, use it here and at render (line 55)